src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -275,7 +276,10 @@ def get_timestamp(group: Group, event: Optional[Event]) -> float:
     return to_timestamp(max(ts, event.datetime) if event else ts)
 
 
-def get_color(event_for_tags: Optional[Event]) -> str:
+def get_color(event_for_tags: Optional[Event], notification: Optional[BaseNotification]) -> str:
+    if notification:
```

Comment:
I don't think you need this because this should only be called for alert notifications.

Reply:
What do you mean? Everything that has `is_message_issue_unfurl` is True hits this code path. Without this, workflow notifications are red.

Reply:
You're right. In that case I think it makes more sense to make the default color "info" on line 286.